global-alert: Visual refresh (#1256)
steelthreads/agds-next: packages/react/src/global-alert/docs/overview.mdx

---
title: Global alert

> description: Global alerts show pressing and high-signal messages, such as system alerts.

Reviewer: ```suggestion
description: Global alerts display prominent service or system wide messages at the top of the screen.
```

> description: Global alerts display prominent service or system wide messages at the top of the screen.

Author: ```suggestion
description: Global ale display prominent service or system wide messages at the top of the screen.
```

</GlobalAlert>
```

> Global alerts are used to display system-wide alerts that are critical for a user to see. They are used to communicate high-signal messages, such as system alerts.

Reviewer: ```suggestion
Global alerts are intrusive alerts that appear at the top of a page. They are used to communicate system or service messages not page or feature messages. Global alerts are not initiated by the user and should persist for a session but can be dismissed by the user. 
```

> Global alerts should be used sparingly due to their prominent nature. They should also be used in coordination with every product team, to ensure it is treated consistently across microfrontends.

Reviewer: ```suggestion
Global alerts should be used sparingly due to their prominent nature and should be coordinated across product teams if used on microfrontend applications.
```

<DoHeading />

- place alerts at the top of the page above the Header
- keep text concise
- explain the problem/issue and (if possible) when it will be resolved
- offer an alternative path to contact or support if possible

> - consider adding a Close button so users can easily dismiss the alert.

Reviewer: ```suggestion
- consider adding a Close button so users can easily dismiss the alert
```

> - use a title to provide a short summary of the alert

Reviewer: ```suggestion
- use a title to provide a short summary of the alert.
```

</GlobalAlert>
```

Global alert is used to display system-wide alerts that are critical for a user to see. They are used to communicate high-signal messages, such as system alerts.

> Due to the prominent nature of these alerts, they should be used sparingly and in coordination with all microfrontend teams.

Reviewer 2: ```suggestion
Due to the prominent nature of these alerts, they should be used sparingly and in coordination with every product team.
```

	</Text>
</GlobalAlert>
```

## Infomation alert

> An `info` version of GlobalAlert is also available, which can be used to display system-wide information messages which are not negative in nature. This could include a general announcement about a feature or policy.

Reviewer 2: ```suggestion
An `info` version of Global alert is also available, which can be used to display system-wide information messages which are not negative in nature. This could include a general announcement about a feature or policy.
```

> This variant is to be used sparingly.

Reviewer 2: ```suggestion
The `info` variant should be used sparingly.
```
